### DISCUSSION TITLE
Need help to enable Sub-GHz for ieee802154_cc13xx_cc26xx

### DISCUSSION THREAD

**Discussion by mkvenkit** [zephyrproject-rtos/zephyr#48943] [Q&A]
I am looking for some guidance on enabling Sub-GHz for CC13xx. 

I am not an expert, but on reading through the CC1352 TRM, it seems like we just need to set the channel correctly. There are two places in the code that needs change it looks like:

1. Channel setup (11-16)?

https://github.com/zephyrproject-rtos/zephyr/blob/master/drivers/ieee802154/ieee802154_cc13xx_cc26xx.c#L81

2. `loDivider` in `CMD_RADIO_SETUP`:

https://github.com/zephyrproject-rtos/zephyr/blob/1342dadc365ee22199e51779185899ddf7478686/drivers/ieee802154/ieee802154_cc13xx_cc26xx.c#L698

Please let me know if you plan to support Sub-GHz for CC13xx in the near future. If not, any guidance you provide in moving forward would be appreciated. Thanks! @bwitherspoon

**Comment by statropy** [comment]
SubGHz support for the CC13xx/CC26xx was added https://github.com/zephyrproject-rtos/zephyr/pull/29598
